### ISSUE TITLE
Pick file from cloud

### ISSUE THREAD

**Issue by manhnt79** [miguelpruivo/flutter_file_picker#130]
I pick file from google drive ( about 40Mb). App does not response with black screen until file download complete.

**Comment by miguelpruivo** [comment]
Yes. That’s a known issue. Because it’s a blocking action that runs on the UI thread. I’ll try to see if there’s anything that can be done here.

**Comment by miguelpruivo** [comment]
This should be fixed right now in the `1.4.0` that is yet on `beta`. Could you please try replacing in your **pubspec.yaml** like so:
```
file_picker:
    git:
      url: https://github.com/miguelpruivo/plugins_flutter_file_picker.git
      ref: beta
```
and let me know if it's working for you? 

Thank you.

**Comment by miguelpruivo** [comment]
[Version 1.4.0](https://pub.dev/packages/file_picker#140) published with a fix for this. Thank you for reporting. 🎉
